plugin/src/main/java/net/dzikoysk/funnyguilds/data/DataModel.java

```diff
@@ -22,8 +23,8 @@ default void validateLoadedData() {
             return;
         }
 
-        Set<String> guilds = GuildUtils.getNames(GuildUtils.getGuilds());
-        Set<String> regions = RegionUtils.getNames(RegionUtils.getRegions());
+        Collection<String> guilds = Entity.names(GuildUtils.getGuilds());
```

Comment:
Po co w sumie teraz zamiast Set robisz Collection skoro i tak Entity.names ci zwraca Set 🤔

Reply:
`List` i `Set` to `Collection` jakby ktoś za chwile chciał zmieniać zwracany typ w Entity.names to nie będzie już trzeba modyfikować tu kodu który jest zależny od tej metody. Już wystąpiła taka jedna zmiana dlatego dałem sobie `Collection`.

Reply:
Z takim rozumowaniem, to można oznaczyć wszystkie zmienne jako `Object`. Oznaczasz jako `Set`, bo explicit wymagasz braku duplikatów